lib/awesome_spawn.rb

```diff
@@ -101,6 +101,29 @@ def run!(command, options = {})
     command_result
   end
 
+  # Execute `command` in a detached manner
+  #   by default the :err, and :out are redirected to `/dev/null`
+  #   by default :pggroup is true (to make a daemon process)
+  #   by default :pggroup_new is true (needed for windows)
+  # @example With normal output
+  #   AwesomeSpawn.run_detached('echo "Hi" > /tmp/out')
+  #
+  # @example With error output as well
+  #
+  #
+  # if a user defines :out or :err, it is assumed they will define both
+  def run_detached(command, options = {})
+    env, command_line, options = parse_command_options(command, options)
+    # maybe add support later for this
+    raise ArgumentError, "options cannot contain :in_data" if options.include?(:in_data)
+
+    options[[:out, :err]] = ["/dev/null", "w"] unless (options.keys.flatten & [:out, :err]).any?
+    options[:pgroup]      = true unless options.key?(:pggroup)
+    options[:pggroup_new] = true unless options.key?(:pggroup_new)
```

Comment:
I think there are typos here and in all the docs s/pggroup/pgroup/

Reply:
Recommend replacing "/dev/null" with `IO::NULL`.

Reply:
looks like it is `new_pgroup` vs `pgroup_new` too. thnx